src/SUMMARY.md

```diff
@@ -22,3 +22,27 @@
 - [Supporting fiat currencies](guide/fiat_currencies.md)
 - [Buying Bitcoin](guide/buy_btc.md)
 - [Fees for End-Users](guide/end-user_fees.md)
+
+---
+
+# Notifications
+
+- [Implementing mobile notifications](notifications/getting_started.md)
+- [Setting up an NDS](notifications/setup_nds.md)
+- [Using webhooks](notifications/using_webhooks.md)
+- [Integrating the plugin](notifications/setup_plugin.md)
+    - [Android](notifications/android_setup.md)
+      - [Setting up the Foreground Service](notifications/android_service.md)
+      - [Adding the Notification Plugin](notifications/android_plugin.md)
+    - [iOS](notifications/ios_setup.md)
+      - [Setting up the Notification Service Extension](notifications/ios_service.md)
+      - [Adding the Notification Plugin](notifications/ios_plugin.md)
+
+---
+
+# Advanced 
+
+- [Adding logging](notifications/logging.md)
+- [Changing default strings](notifications/changing_strings.md)
+- [Customising push messages](notifications/custom_messages.md)
+- [Handling custom notifications](notifications/custom_notifications.md)
```

Comment:
⚠️ Needs revision from other parties.

There's been one thing bothering me about titles, differences in cases. 
My suggestion is to be consistent and use Title Case on all these sections.
```suggestion
- [Implementing Mobile Notifications](notifications/getting_started.md)
- [Setting Up an NDS](notifications/setup_nds.md)
- [Using Webhooks](notifications/using_webhooks.md)
- [Integrating the Plugin](notifications/setup_plugin.md)
    - [Android](notifications/android_setup.md)
      - [Setting Up the Foreground Service](notifications/android_service.md)
      - [Adding the Notification Plugin](notifications/android_plugin.md)
    - [iOS](notifications/ios_setup.md)
      - [Setting Up the Notification Service Extension](notifications/ios_service.md)
      - [Adding the Notification Plugin](notifications/ios_plugin.md)

---

# Advanced 

- [Adding Logging](notifications/logging.md)
- [Changing Default Strings](notifications/changing_strings.md)
- [Customising Push Messages](notifications/custom_messages.md)
- [Handling Custom Notifications](notifications/custom_notifications.md)
```

Reply:
I understand, but I'm not going to change the titles as they are approved for Greenlight SDK and Roy was very particular about it